lib/scala/kg/src/main/scala/data/kg/KgtkDataResource.scala

```diff
@@ -0,0 +1,19 @@
+package data.kg
+
+import java.io.InputStream
+
+import data.DataResources
+import formats.kg.kgtk.KgtkTsvReader
+import io.github.tetherlessworld.twxplore.lib.base.WithResource
+import models.kg.{KgEdge, KgNode}
+
+class KgtkDataResource(val tsvResourceName: String) extends DataResources with WithResource {
+  def getTsvResourceAsStream(): InputStream =
+    getResourceAsStream(tsvResourceName)
+
+  def read(): Tuple3[List[KgEdge], List[KgNode], List[KgNode]] = {
```

Comment:
Not a fan of using tuples as types, since you then have to remember which position corresponds to which semantics if you have duplicate-typed fields, as you do here. Please use a case class instead.

Reply:
Changed to `List[KgEdgeWithNodes]`